AncientReligions/events/on_action_events.txt

```diff
@@ -20,6 +20,530 @@ character_event = {
 	desc = "B"
```

Comment:
I didn't change anything, it was already like that ^^
Do you know if it is possible to override an event by redefining it in another file loaded after, with the same id ?
Or maybe cleaner, make a maintenance event that triggers for the recently activated head of reformed religion, and sets the correct flag ?